panacea/grpc_client.go

```diff
@@ -123,6 +123,20 @@ func (c *GrpcClient) GetChainId() (string, error) {
 	return response.GetBlock().GetHeader().ChainID, nil
 }
 
+func (c *GrpcClient) GetRegisteredDataValidator(address string) (datapooltypes.DataValidator, error) {
+	client := datapooltypes.NewQueryClient(c.conn)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	res, err := client.DataValidator(ctx, &datapooltypes.QueryDataValidatorRequest{Address: address})
+	if err != nil {
+		return datapooltypes.DataValidator{}, err
+	}
+
+	return *res.GetDataValidator(), err
```

Comment:
I think it's better to return nil for errors.
Of course, `err` is always nil, but it seems better to declare it expilicitly.
```suggestion
	return *res.GetDataValidator(), nil
```

Reply:
Oh, My mistake. I'll fix it.
> but it seems better to declare it expilicitly
And I'll add `fmt.Errorf` too. Thanks